x-pack/plugins/monitoring/server/alerts/memory_usage_alert.ts

```diff
@@ -0,0 +1,343 @@
+        values: {
+          nodeName: stat.nodeName,
+          memoryUsage: stat.memoryUsage,
+        },
+      }),
+      nextSteps: [
+        createLink(
+          i18n.translate('xpack.monitoring.alerts.memoryUsage.ui.nextSteps.tuneThreadPools', {
+            defaultMessage: '#start_linkTune thread pools#end_link',
+          }),
+          `{elasticWebsiteUrl}guide/en/elasticsearch/reference/{docLinkVersion}/modules-threadpool.html`
+        ),
+        createLink(
+          i18n.translate('xpack.monitoring.alerts.memoryUsage.ui.nextSteps.managingHeap', {
+            defaultMessage: '#start_linkManaging ES Heap#end_link',
+          }),
+          `{elasticWebsiteUrl}blog/a-heap-of-trouble`
+        ),
+        createLink(
+          i18n.translate('xpack.monitoring.alerts.memoryUsage.ui.nextSteps.identifyIndicesShards', {
+            defaultMessage: '#start_linkIdentify large indices/shards#end_link',
+          }),
+          'elasticsearch/indices',
+          AlertMessageTokenType.Link
+        ),
+        createLink(
+          i18n.translate('xpack.monitoring.alerts.memoryUsage.ui.nextSteps.addMoreNodes', {
+            defaultMessage: '#start_linkAdd more data nodes#end_link',
+          }),
+          `{elasticWebsiteUrl}guide/en/elasticsearch/reference/{docLinkVersion}/add-elasticsearch-nodes.html`
+        ),
+        createLink(
+          i18n.translate('xpack.monitoring.alerts.memoryUsage.ui.nextSteps.resizeYourDeployment', {
+            defaultMessage: '#start_linkResize your deployment (ECE)#end_link',
+          }),
+          `{elasticWebsiteUrl}guide/en/cloud-enterprise/current/ece-resize-deployment.html`
+        ),
+      ],
+      tokens: [
+        {
+          startToken: '#absolute',
+          type: AlertMessageTokenType.Time,
+          isAbsolute: true,
+          isRelative: false,
+          timestamp: alertState.ui.triggeredMS,
+        } as AlertMessageTimeToken,
+        {
+          startToken: '#start_link',
+          endToken: '#end_link',
+          type: AlertMessageTokenType.Link,
+          url: `elasticsearch/nodes/${stat.nodeId}`,
+        } as AlertMessageLinkToken,
+      ],
+    };
+  }
+
+  protected executeActions(
+    instance: AlertInstance,
+    { alertStates }: AlertInstanceState,
+    item: AlertData | null,
+    cluster: AlertCluster
+  ) {
+    const firingNodes = alertStates.filter(
+      (alertState) => alertState.ui.isFiring
+    ) as AlertMemoryUsageState[];
+    const firingCount = firingNodes.length;
+
+    if (firingCount > 0) {
+      const shortActionText = i18n.translate('xpack.monitoring.alerts.memoryUsage.shortAction', {
+        defaultMessage: 'Verify memory usage levels across affected nodes.',
+      });
+      const fullActionText = i18n.translate('xpack.monitoring.alerts.memoryUsage.fullAction', {
+        defaultMessage: 'View nodes',
+      });
+
+      const action = `[${fullActionText}](elasticsearch/nodes)`;
```

Comment:
This link doesn't work from an email

Reply:
Doesn't look like we're using these links in the email template, and I can't find it in the code. Or, am I missing something?